### DISCUSSION TITLE
Make sandboxie-plus have a similar function to virtual machine suspend

### DISCUSSION THREAD

**Discussion by hiwo11** [sandboxie-plus/Sandboxie#1928] [Various ideas and suggestions]
Whether it is possible to make SandBoxie-Plus have similar functions that are similar to virtual machines, so that we can save the software status and save a variety of states. When we need it, we can quickly recover, such as entertainment status, programming status.  This function is also similar to the dormant function of Windows, but contains a variety of states. This is not a Windows dormant that can be achieved

**Comment by DavidXanatos** [comment]
No it is not possible in any way to save the state of a random application terminate it and then re start it and restore the state, all handles would be invalid after such an operation. 
Microsoft would have to re-architecture the NT kernel to allow for that.
